mailet/scala/pom.xml

```diff
@@ -0,0 +1,93 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+  ~ Licensed to the Apache Software Foundation (ASF) under one  ~
+  ~ or more contributor license agreements.  See the NOTICE file~
+  ~ distributed with this work for additional information       ~
+  ~ regarding copyright ownership.  The ASF licenses this file  ~
+  ~ to you under the Apache License, Version 2.0 (the           ~
+  ~ "License"); you may not use this file except in compliance  ~
+  ~ with the License.  You may obtain a copy of the License at  ~
+  ~                                                             ~
+  ~   http://www.apache.org/licenses/LICENSE-2.0                ~
+  ~                                                             ~
+  ~ Unless required by applicable law or agreed to in writing,  ~
+  ~ software distributed under the License is distributed on an ~
+  ~ "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY      ~
+  ~ KIND, either express or implied.  See the License for the   ~
+  ~ specific language governing permissions and limitations     ~
+  ~ under the License.                                          ~
+  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~-->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>org.apache.james</groupId>
+        <artifactId>apache-mailet</artifactId>
+        <version>3.6.0-SNAPSHOT</version>
+        <relativePath>../</relativePath>
+    </parent>
+
+    <artifactId>apache-mailet-scala</artifactId>
+    <packaging>jar</packaging>
+
+    <name>Apache James :: Scala Mailets</name>
+    <description>Apache James Scala Mailets is a small collection of general purpose mailets
+        with limited dependencies. These mailets can be used in any mailet container.</description>
+
+    <properties>
+        <encoding>UTF-8</encoding>
+        <spec2.version>4.9.4</spec2.version>
+    </properties>
+
+    <dependencies>
+        <dependency>
+            <groupId>${project.groupId}</groupId>
+            <artifactId>apache-mailet-test</artifactId>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>${project.groupId}</groupId>
+            <artifactId>james-server-dnsservice-test</artifactId>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>${project.groupId}</groupId>
+            <artifactId>james-server-mailets</artifactId>
+        </dependency>
```

Comment:
That dependancy to a server project in a mailet project is evil.

Reply:
in fact this mailet depends on an abstract mailet from server.
I move the scala mailets into server to be able to keep this dependency